source/clear-linux/reference/collaboration/documentation/inline.rst

```diff
@@ -34,20 +34,29 @@ the roles.
 
```

Comment:
Hello @intelkevinputnam.
Starting at Line 21, it appears the legacy version of this doc had several alignment errors. While you're at it, would you mind realigning indents Line 21 and thereafter to fit cascading logic?